index.html

```diff
@@ -9,221 +9,295 @@
+      </div>
+      <br />
+      <ul>
+        <li class="frameworks">
+          <hr />
+          <br />
         </li>
-        <li>
-          <a href=""><img src="assets/image/social_icon/github.png" class="headline-icon" alt="github"></a>
+      </ul>
+      <ul>
+        <li class="skills">
+          <div class="user-tool-d">
+            <h3>Skills</h3>
+            <i class="fas fa-chevron-right"></i>
+          </div>
+          <br />
+          <hr />
+
         </li>
       </ul>
-    
-      <button class="about-button"  type="button">Get my resume</button>
-      <div class="language-section">
-        <ul class="lang">
-          <li class="lang-exp">
-            <div class="no-1">
-              <p>Languages</p>
-              <img src="assets/image/arrow_bottom.png" alt="arrow_bottom" />
-            </div>
-            <ul class="languages">
-              <li>
-                <img src="assets/image/js_logo.png" alt="JavaScript" />
-                <a href="#">JavaScript</a>
-              </li>
-              <li>
-                <img src="assets/image/htmlLogo.png" alt="Html" />
-                <a href="#">HTML</a>
-              </li>
-              <li>
-                <img src="assets/image/css_logo.png" alt="CSS" />
-                <a href="#">CSS</a>
-              </li>
-            </ul>
+    </div>
+  </section>
+  <section class="contact-form">
+    <div class="contact-form-holder">
+      <h2 class="contact-title">Contact me</h2>
+
+      <p class="p-pargraph">
+        If you have an application you are interested in developing, a feature
+        that you need built or a project that needs coding. I’d love to help
+        with it
+      </p>
+
+      <form action="https://formspree.io/f/xqkwonvv" class="form-contact" method="post">
+        <ul class="ul-form">
+          <li class="li-form">
+            <input type="text" placeholder="Yeremias" name="name" id="name_form" maxlength="30" required />
           </li>
-          <li class="lang-exp">
-            <div class="no-1">
-              <p>Frameworks</p>
-              <img src="assets/image/arrow_bottom.png" alt="arrow_bottom" />
-            </div>
-            <hr />
+          <li class="li-form">
+            <input type="email" placeholder="natajayanj14@gmail.com" name="email" id="email_form" />
           </li>
-
-          <li class="lang-exp">
-            <div class="no-1">
-              <p>Skills</p>
-              <img src="assets/image/arrow_bottom.png" alt="arrow_bottom1" />
-            </div>
-            <hr />
+          <li class="li-form">
+            <textarea name="message" placeholder="Write your message here" id="message" cols="30" rows="10"
+              maxlength="500" required></textarea>
+          </li>
+          <li class="li-form">
+            <input id="get-submit" type="submit" value="Get It Touch" class="btn-in" />
           </li>
         </ul>
-      </div>
+      </form>
     </div>
```

Comment:
- [ ] As required for the project, kindly make sure your design stick as much as possible to the one from your chosen Figma template e.g.: background,...

| Figma Temaplate | Your Design |
| ----------------------- | ----------------- |
| ![Screenshot from 2023-04-04 09-48-23](https://user-images.githubusercontent.com/83042742/229816932-53e10c00-e2f9-48fd-a8be-5890e981d2ad.png) | ![Screenshot from 2023-04-04 09-49-33](https://user-images.githubusercontent.com/83042742/229816997-f8ce587f-6422-480c-9501-69881f1e5102.png) |

Reply:
Resolve